server/src-lib/Hasura/Metadata/Class.hs

```diff
@@ -32,38 +34,49 @@ runMetadataStorageT =
 
 {- Note [Todo: Common interface for eventing sub-system]
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Scheduled and Table event triggers have their own implementation
-and tables in Postgres. The core logic for both is same. So it
-is necessary to have a unified interface (maybe via a Postgres extension)
-where all event triggers implementation can extend from. By this we can
-fairly reduce the Postgres schema foot print and interactions to the database.
+Postgres tables' event triggers and scheduled event triggers are similar in the
+core logic. But currently, their implementation is completely isolated and do not share
+a common schema in Postgres. We're having a plan to simplify them via a common interface
+(maybe via a Postgres extension). This will potentially reduce number of interactions made
+to database and schema foot print.
```

Comment:
Thanks for adding this note. Could you specify in this note what code it is referring to, e.g. which methods would need to get changed in this plan? How would it reduce the number of interactions made to the database, and how would it reduce the schema footprint?

Reply:
All these questions will be answered in an RFC for the same. I'm not sure about specifically what methods will be changed/dropped. One thing we're sure that the common interface would simplify things related to eventing sub-system. 

cc @0x777